simde/arm/neon/ld1.h

```diff
@@ -42,7 +43,11 @@ simde_vld1_f16(simde_float16_t const ptr[HEDLEY_ARRAY_PARAM(4)]) {
     return vld1_f16(ptr);
   #else
     simde_float16x4_private r_;
-    simde_memcpy(&r_, ptr, sizeof(r_));
+    #if defined(SIMDE_RISCV_V_NATIVE) && SIMDE_ARCH_RISCV_ZVFH
+      r_.sv64 = __riscv_vle16_v_f16m1((_Float16 *)ptr , 4);
+    #else
+      simde_memcpy(&r_, ptr, 8);
```

Comment:
Can the `sizeof` version stay in the non-`RISCV_V` branches?

Is that not working for the native RVV types a compiler bug?

Reply:
Yes, sizeof can remain in the non-RVV branch. I will modify them.

Reply:
There are no bugs when compiling RVV types. However, the size of unions in type.h, such as simde_xxx_private, may not be as expected. The size of these unions can vary due to the differing lengths of RVV vector machines.